LEAF_Nexus/admin/templates/site_elements/generic_plainDialogLarge.tpl

```diff
@@ -15,5 +15,7 @@ $(function() {
 		width: $(window).width() * 0.8,
 		height: $(window).height() * 0.8
 	});
+    $('#plainDialogLarge').dialog({minWidth: ($(window).width() * .8) + 30});
```

Comment:
Did routing require changing the style of a dialog?

Reply:
This has dev merged into it. So there are some changes not related to routing.